src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Web.ViewModels.Apply;
+
+namespace SFA.DAS.EmployerIncentives.Web.Controllers
+{
+    [Route("{accountId}/[Controller]")]
+    public class ApplyController : Controller
+    {
+        [Route("")]
+        [HttpGet]
+        public async Task<ViewResult> QualificationQuestion()
+        {
+            return View(new QualificationQuestionViewModel());
+        }
+
+        [Route("")]
+        [HttpPost]
+        public async Task<IActionResult> QualificationQuestion(string accountId, QualificationQuestionViewModel viewModel)
+        {
+            if (!viewModel.HasTakenOnNewApprentices.HasValue)
+            {
+                ModelState.AddModelError("HasTakenOnNewApprentices", "Select yes if you’ve taken on new apprentices that joined your payroll after 1 August 2020");
```

Comment:
Can this error message go into a resource or constants file?

Reply:
What's the value of having them in a resource or constants file?

Reply:
It's an approach used across most other ESFA repos I've encountered, you don't have magic strings in unit tests, only one place to update the content if/when wording changes, plus SonarCube won't moan at you ;)